plugins/org/src/components/Cards/OwnershipCard/ComponentsGrid.tsx

```diff
@@ -44,8 +44,16 @@ const useStyles = makeStyles((theme: BackstageTheme) =>
       },
       height: '100%',
     },
-    bold: {
+    title: {
       fontWeight: theme.typography.fontWeightBold,
+      textAlign: 'center',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      maxWidth: '100%',
+      '&:hover': {
+        overflow: 'visible',
+        wordBreak: 'break-word',
+      },
```

Comment:
Another option for the hover reveal is to use the [`OverflowTooltip`](https://backstage.io/storybook/?path=/story/data-display-overflowtooltip--default) component from `@backstage/core-components`. Not sure which one will provide the best UX in this case but could be worth trying it out.

Reply:
Awesome, I didn't know about `OverflowTooltip` this is great!! I refactored to use that component instead, and also added in a few comments to the PR. I think this looks better, but not 100% confident from a UX standpoint.